metadata-models/src/main/pegasus/com/linkedin/identity/CorpUserHomePageSettings.pdl

```diff
@@ -9,5 +9,10 @@ record CorpUserHomePageSettings {
   /**
    * The page template that will be rendered in the UI by default for this user
    */
-  pageTemplate: Urn
+  pageTemplate: optional Urn
+
+   /**
+   * The list of announcement urns that have been dismissed by the user
+   */
+  dismissedAnnouncements: optional array [string]
```

Comment:
i think it makes sense to explicitly type this as an `Urn` here since we know these will be announcement urns that are dismissed